### ISSUE TITLE
[ISSUE] with `databricks_user` resource. Adding 'force' for admin user fails

### ISSUE THREAD

**Issue by rohitnijhawan** [databricks/terraform-provider-databricks#1639]
- The force flag was manually added to the users.tf file output after the exporter ran.
- User data_lake is an admin in the source workspace
- User data_lake and their admin token is used to run terraform apply on the target workspace

### Configuration
```hcl
  resource "databricks_user" "data_lake" {
    user_name                  = "data_lake@acmecorp.com"
    allow_instance_pool_create = true
    allow_cluster_create       = true
    active                     = true
    force                      = true
  }
```

### Expected Behavior
We expected that the existence of the user would be ignored as per the documentation.

### Actual Behavior
- The Terraform apply command issues an error:
The user [data_lake@acmecorp.com] already exists in the DR workspace. 
- We see below error message
Error: cannot delete group member: PERMISSION_DENIED: Requesting user [data_lake@acmecorp.com] is not an admin.
- And also, attempting to insert **force = true** removed  “Admin” permissions of that user in the destination workspace.

### Steps to Reproduce
Run terraform exporter
Run terraform apply against a different workspace multiple times using the token of an admin user.

### Terraform and provider versions
TF > 0.15
Provider > 1.2.0

**Comment by alexott** [comment]
Hmmm, I'm not able to reproduce it. Please collect debug logs when this happens

**Comment by alexott** [comment]
let's track it as a part of #2648

**Comment by alexott** [comment]
Fixed by #2878
